client/state/signup/steps/website-content/schema.ts

```diff
@@ -0,0 +1,55 @@
+export const schema = {
+	type: 'object',
+	patternProperties: {
+		homePage: { content: 'string', imageUrl1: 'string', imageUrl2: 'string', imageUrl3: 'string' },
```

Comment:
In the near future, there might be these two changes to the product:
1. We'll allow users to change the title of their pages.
2. We'll allow users to add more pages.

Also, the 4th-page title will depend on the vertical selected in the design picker (similar to our previous logic in Typeform)

So can we make this schema a bit more flexible? For example:
```
pages: [
				{
					title: 'Home',
					content: 'Text content entered by the user',
					images: [
						'url1', 'url2', 'url3'
					]
				},
				{
					title: 'About',
					content: 'Text content entered by the user',
					images: [
						'url1', 'url2', 'url3'
					]
				},
				{
					title: 'Contact',
					content: 'Text content entered by the user',
					images: [
						'url1', 'url2', 'url3'
					]
				}
			]
```
Related Slack thread: p1642141995001800-slack-C02KVCAL7GX

Reply:
Ah yes nice catch I will fix this!